### DISCUSSION TITLE
Oracle-server error connection to XXX_DEV: NJS-516: no configuration directory set or available to search for tnsnames.ora

### DISCUSSION THREAD

**Discussion by tlraghavendra** [vtulluru/node-red-contrib-oracledb-mod#48] [Q&A]
Hi,

After I updated to latest version I am getting error, any advise to clear this error?

![2024-01-19 14_40_25](https://github.com/vtulluru/node-red-contrib-oracledb-mod/assets/5299237/c66aa6d2-5bcd-4406-afe3-73e9cf4dc903)

Screenshot showing path is wrong as i was testing
actual i am using "C:\oracle\product\12.2.0\client_2"

Node-RED version: v3.1.3
Node.js  version: v20.11.0
Windows_NT 10.0.14393 x64 LE

**Comment by w3ori** [comment]
I have the same problem, created an issue.
Last working version was 0.6.3 so i downgraded to that.
